### PR TITLE
task 7. setting up of api gateway basic authorization

### PR DESCRIPTION
**Pull request by petr150894**
**Link to Application** - https://d3pu3r4179y896.cloudfront.net/admin/products

**Import product endpoint protected by basicAuthorizer:**
  GET - https://9nhhji66k6.execute-api.eu-west-1.amazonaws.com/develop/import/products

**FE PR** - https://github.com/petr150894/garage-store-nodejs-aws-fe/pull/5

**Evaluation criteria:**
1 - authorization-service is added
3 - import-service serverless.yaml file has authorizer configuration for the importProductsFile lambda.  Generating 403 and in 401 HTTP statuses implemented correctly
5 - client application updated to send Authorization: Basic authorization_token header on import

**Additional tasks:**
+1 - Client application displays alerts for the responses in 401 and 403 HTTP statuses.

### CODE REVIEW

**Comment by BaturevichNikita** [comment]
I can't check, because the authorization token doesn't work properly. I used username =  petr150894 and password = TEST_PASSWORD. Am I right?

**Comment by maximsan** [comment]
I has the same problem. Could you add more details about how to test you app ? Maybe it will help, because It seems like something is not clear. Thanks!

**Comment by petr150894** [comment]
Sorry, my mistake. I used another password and forgot to write it in the description
You go here - https://www.base64encode.org and encode this string 'petr150894:ADMIN_PASSWORD', (or just use this token - cGV0cjE1MDg5NDpBRE1JTl9QQVNTV09SRA==)
Then you paste it into the local storage by this way and everything works properly 


image

@BaturevichNikita  @maximsan

File: authorization-service/config.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import * as dotenv from 'dotenv';
+
+dotenv.config({ path: '.env.default' });
+
+export default {
+  petr150894: process.env.petr150894
```
Comment (maximsan) [inline authorization-service/config.ts:6]: Actually it's extra work if you don't use it anywhere else, serverless-dotenv-plugin upload env variables automatically from .env file

**Review by maximsan — APPROVED** [review]
It looks working. 6/6

**Pull request merged**